packages/desktop-electron/package.json

```diff
@@ -3,13 +3,13 @@
   "author": "Actual",
   "productName": "Actual",
   "description": "A simple and powerful personal finance system",
-  "version": "24.8.0",
+  "version": "24.8.3",
```

Comment:
Every time we make changes to the store listing we need to increment - this means we may go out of sync with the main package